src/pykeen/models/base.py

```diff
@@ -19,7 +19,7 @@
 from ..tqdmw import tqdm
 from ..triples import TriplesFactory
 from ..typing import MappedTriples
-from ..utils import NoRandomSeedNecessary, get_embedding, resolve_device, set_random_seed
```

Comment:
Is the get_embedding function not used anymore? Maybe it should delegate to the new code and emit a deprecation warning

Reply:
It is used in a few places.